### PR TITLE
:sparkles: Add cast portrait element

### PR DESCRIPTION
**Pull request by J-Bannach**
Element consists of image with vignette effect and cast name

### CODE REVIEW

**Review by daebert — COMMENTED** [review]
Good work!

**Review by Dev-dfm — COMMENTED** [review]
lgtm 👍

**Review by SGierszewski — COMMENTED** [review]
Well done!

**Comment by dhnazarenko** [comment]
lgtm 🚀

File: src/app/components/CastPortrait/CastPortrait.module.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  z-index: -1;
+}
+
+.castPortrait__firstName,
+.castPortrait__lastName {
+  text-align: center;
+  margin: 0;
+  font-family: var(--font-family);
+
+  /* position: absolute; */
```
Comment (phlgr) [inline src/app/components/CastPortrait/CastPortrait.module.css]: Please remove this comment

**Review by JGieseWeb — APPROVED** [review]

**Pull request merged**